cluster-monitoring/values.yaml

```diff
@@ -0,0 +1,421 @@
+# prometheus-operator:
```

Comment:
Because if you don't set your own values the defaults should come from the subcharts. I think it's overkill of having to maintain these values every time something in the dependencies changes.